src/test/java/seedu/address/model/person/UniquePersonListTest.java

```diff
@@ -3,11 +3,11 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_ADDRESS_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_HUSBAND;
+import static seedu.address.logic.commands.CommandTestUtil.ADDRESS_BEN;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_GROUP_1;
 import static seedu.address.testutil.Assert.assertThrows;
 import static seedu.address.testutil.TypicalPersons.ALBERT;
-import static seedu.address.testutil.TypicalPersons.BOB;
+import static seedu.address.testutil.TypicalPersons.BEN;
```

Comment:
Have to standardise here. Either BEN or BART.

Reply:
Originally, it used to be Amy and Bob, and Alice and Benson defined in the same way. So I just did a one-for-one mapping between them. Should we consider just making them the same person (one person for A and one person for B)?

Reply:
If there is one-to-one mapping of names with no discrepancies for variable name WITH String text, then I am ok with it. Would you be making the changes?